modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -39,6 +40,29 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.12.1
+
+Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.12.1 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
+
+=== Prevent 507 Insufficient Storage on 32-Bit Systems
+
+With the introduction of https://github.com/owncloud/core/pull/40567 compatibility with 32-bit systems broke as we are now casting $freeSpace to int and this caused an integer overflow on such systems when the free space was above the max supported value. We added therefore an additional check for 32-bit systems in QuotaPlugin.php. https://github.com/owncloud/core/pull/40709[#40709]
+
+=== User-Key Encryption Is No Longer Auto-Enabled
+
+Executing the "encryption:encrypt-all" occ command will no longer auto-enable user-key encryption. https://github.com/owncloud/core/pull/40702[#40702]
```

Comment:
Maybe also add a note reminding readers that user-key encryption has been deprecated (since oC 10.7 - I think that is correct).

Reply:
See docs-server PR https://github.com/owncloud/docs-server/pull/912